source/scene.py

```diff
@@ -0,0 +1,107 @@
+from kivy.graphics import *
+
+from ball import Ball
+from hostile_balls_spawner import HostileBallsSpawner
+from physics_engine import PhysicsEngine
+from player import Player
+from vector import Vec2D
+
+
+class Scene:
+
+    def __init__(self, parent):
+        from scene_data import SceneData
+        self.data = SceneData()
+        self.player = None
+        self.engine = PhysicsEngine(self)
+        self.parent_widget = parent
+        self.__ball_spawn_dt = 0.0
+        self.__last_touch = 0.0
+        self.dt = 0.0
```

Comment:
self.dt is and will always be used only inside the Scene objects. Therefore it should be private, especially since self.__ball_spawn_dt and self.__last_touch already are private